dps_client/adapters/iothub_security_sas.c

```diff
@@ -469,7 +469,7 @@ BUFFER_HANDLE iothub_security_sas_sign_data(SECURITY_DEVICE_HANDLE handle, const
         BYTE data_signature[1024];
         BYTE* data_copy = (unsigned char*)data;
         /* Codes_SRS_SECURE_DEVICE_TPM_07_021: [ iothub_security_sas_sign_data shall call into the tpm to hash the supplied data value. ] */
-        uint32_t sign_len = SignData(&handle->tpm_device, &NullPwSession, data_copy, data_len, data_signature, sizeof(data_signature) );
+        uint32_t sign_len = SignData(&handle->tpm_device, &NullPwSession, data_copy, (uint32_t)data_len, data_signature, (uint32_t)sizeof(data_signature) );
```

Comment:
>(uint32_t)data_len, data_signature, (uint32_t)sizeof(data_signature) [](start = 85, length = 68)

I assume this change is not related to removing the WIP

Reply:
+1.  Also adding soapbox comment about having 2 unrelated changes in a single commit and how that can complicate source code management.

---
In reply to: [138970052](https://github.com/Azure/azure-iot-sdk-c/pull/231#discussion_r138970052) [](ancestors = 138970052)

Reply:
Yea, yea.  Ok, you have made your point, I'll move this into to commits.

---
In reply to: [138972637](https://github.com/Azure/azure-iot-sdk-c/pull/231#discussion_r138972637) [](ancestors = 138972637,138970052)